.gitignore

```diff
@@ -414,3 +414,4 @@ FodyWeavers.xsd
 # AppSettings Development file
 **/appSettings.Development.json
 /src/EPR.ProducerContentValidation.FunctionApp/localSettings.json
+/src/EPR.ProducerContentValidation.FunctionApp/local.settings 18.json
```

Comment:
Do you need this line? If you have to have multiple localsettings files, you should be able to just change the line above  to 
```
/src/EPR.ProducerContentValidation.FunctionApp/localSettings*.json
```

Reply:
local.settings 18.json has been removed from .gitignore